devtools/run_doctest.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+if [ $TRAVIS_PYTHON_VERSION == '3.7' ]; then
+  find ./deepchem -name "*.py" ! -name '*load_dataset_template.py' | xargs python -m doctest -v;
+fi
```

Comment:
Splitting these out into separate scripts is risky.  We periodically update what python versions we run tests on.  In a year or two we may not be running on 3.7 anymore so this check will need to be updated.  But if it's hidden away in a separate file, there's a good chance no one will think to look there and so the doctests will silently stop getting run.

Reply:
Thanks, I agree with your comment! I stop splitting